src/mame/konami/firebeat.cpp

```diff
@@ -2660,15 +2700,17 @@ GAMEL( 2000, kbm2nd, 0,   firebeat_kbm, kbm, firebeat_kbm_state, init_kbm_jp, RO
 GAMEL( 2001, kbm3rd, 0,   firebeat_kbm, kbm, firebeat_kbm_state, init_kbm_jp, ROT270, "Konami", "Keyboardmania 3rd Mix", MACHINE_IMPERFECT_SOUND, layout_firebeat )
 
 // Requires DVD CHD support. Once DVD CHD support is implemented then MACHINE_NOT_WORKING can be removed
```

Comment:
Should this comment be updated now that it's been done?

Reply:
Good catch, completely missed that. Fixed.